Update 10.peptide-ionization.md (#10)
jessegmeyerlab/proteomics-tutorial: content/10.peptide-ionization.md

## Peptide Ionization {.page_break_before}

> Before MALDI and ESI[@DOI:10.1126/science.2675315], the so-called soft-ionization techniques were developed, the analysis of peptides was a challenge for mass spectrometry. 

Reviewer: ```suggestion
Before the year 1990, peptides analysis by mass spectrometry was challenging. 
```

Reviewer: not sure the exact year to use here?

> It was a big ask to introduce ionized peptides using a hard ionization technique (like fast atom bombardment) into the mass spectrometer, without destroying or breaking them.

Reviewer: ```suggestion
Hard ionization techniques in use at the time, like fast atom bombardment, were not directly applicable to peptides without destroying or breaking them.
```

> It was a big ask to introduce ionized peptides using a hard ionization technique (like fast atom bombardment) into the mass spectrometer, without destroying or breaking them.

Reviewer: can we cite early work on how people were trying to do FAB on peptides?

Author: Going deeper into FAB may wararnt other ionization techniques explanations like electron ionization (EI), chemical ionization (CE) etc. So, I think it's better to focus on ESI and MALDI.

The soft ionization tehniques however, revolutionized the proteomics field and it became possible to routinely ionize and analyze peptides using MALDI and ESI techniques at high-throughput scale. 

> Such was the impact of these two techniques that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules". [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].

Reviewer: ```suggestion
These two techniques were so impactful that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules". [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
```

### MALDI
In this method, the in-gel digested protein sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound (like cyno-hydroxycinnamic acid).

> Subsequently, the analyte is placed in a vacuum chamber in which it is subjected to a 337nm LASER.

Reviewer: ```suggestion
Subsequently, the analyte is placed in a vacuum chamber in which it is irradiated with a LASER, usually at 337nm [@DOI:10.1021/cr010375i].
```

> In this method, the in-gel digested protein sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound (like cyno-hydroxycinnamic acid).

Reviewer: can we cite a review of different matrices that are possible?

Reviewer: and also a general review of MALDI since you go into the mechanism here

Author: Do I also cover that MALDI was actually developed by Karas and Hillenkamp, but not considered for nobel, while its their  version that survives, not the one from Tanaka.

Reviewer: I think it would be great to cover that, maybe in a separate paragraph. I would prefer to discuss the controversy without taking sides since I don't know the facts well.

Author: > can we cite a review of different matrices that are possible?

Done

Author: > and also a general review of MALDI since you go into the mechanism here

Added

Author: > I think it would be great to cover that, maybe in a separate paragraph. I would prefer to discuss the controversy without taking sides since I don't know the facts well.

Covered that in short. See if it fits.

> In this method, the in-gel digested protein sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound (like cyno-hydroxycinnamic acid).

Reviewer: ```suggestion
In this method, the peptide sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound (like alpha-cyano-hydroxycinnamic acid).
```

Reviewer: could be a normal digest not necessarily in-gel

Author: Agreed

> This laser energy is just about right to ionize the matrix and peptides without breaking them.

Reviewer: ```suggestion
This laser energy is absorbed by the matrix, which then transfers that energy along with its free protons to the co-crystalized peptides without significantly breaking them.
```

> Subsequently, the analyte is ionized by Matrix-assisted laser desorption ionization (MALDI), in which it is subjected to a 337nm LASER.

Reviewer: ```suggestion
Subsequently, the analyte is placed in a vacuum chamber in which it is subjected to a 337nm LASER.
```

Reviewer: is this the only laser wavelength in use?

Author: I will need to check, this is the one used earlier. Don't know if it changed over the years

> This laser energy is just about right to ionize the matrix and peptides without breaking them.

Reviewer: I think this sentence is trying to comment on how much energy is used? can we be more specific and include citation(s)?

Author: More than energy, its hinting towards the volatile matrix bearing the brunt of energy to dissipate their protons which effectively protects the peptides from breaking, ratio is 1000:1 :: matrix:peptides

Reviewer: I like the way you phrase this comment - can you rephrase like that?

The matrix and co-crystallized sample generate plumes, and the volatile matrix imparts its protons to the peptides as it gets ionized first.
The weak acidic conditions used as well as the acidic nature of the matrix allows easy exchange of protons for the peptides to get ionized.

> These ionized peptides generally form the metastable ions, most of them will fragment quickly.

Reviewer: please cite

However, it can take several milliseconds and the mass spectrometry analysis can be performed before this time.

### Electrospray Ionization

> In the electrospray ionization, reverse-phase liquid-chromatography setup is usually connected to an online nano-ESI source.

Reviewer: please add citation(s)

Author: added the main citation from J. Fenn.

A high voltage is applied between the spraying needle and the mass spectrometer.
Through this needle, a fine mist of analyte as liquid droplets form at the needle.
Due to voltage and pressure, the droplet bursts into plumes of ions carrying protons from the liquid phase of the chromatography to impart ions to the analytes.

> The liquid phase is generally kept acidic to impart protons easily to the analytes.

Reviewer: should mention taylor cone and coulombic explosion somewhere in this part, These look like a good overviews to cite: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3348530/ 
https://doi.org/10.1002/(SICI)1096-9888(199707)32:7%3C677::AID-JMS536%3E3.0.CO;2-G

Author: Looks fine to cite these two.

## Peptide Ionization {.page_break_before}
Until the early 1990s, peptides analysis by mass spectrometry was challenging. 
Hard ionization techniques in use at the time, like fast atom bombardment, were not directly applicable to peptides without destroying or breaking them.
The soft ionization tehniques however, revolutionized the proteomics field and it became possible to routinely ionize and analyze peptides using MALDI and ESI techniques at high-throughput scale. 
These two techniques were so impactful that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules". [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/]. 
### MALDI
The term, Matrix-assisted LASER desorption ionization (MALDI), was coined by Hillenkamp and Karas in 1985[@URL:https://pubs.acs.org/doi/abs/10.1021/ac00291a042]. 

> Karas and Hillenkamp discovered MALDI technique first, although a similar ionization method was shown by Koichi Tanaka in 1988 [@URL:https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/10.1002/rcm.1290020802]. Few months later, Karas and Hillenkamp also demonstrated MALDI applied to protein ionization[@URL:https://pubs.acs.org/doi/abs/10.1021/ac00171a028].

Reviewer: ```suggestion
Karas and Hillenkamp discovered the MALDI technique first, although a similar ionization method was shown by Koichi Tanaka in 1988 [@DOI:10.1002/rcm.1290020802]. 
A few months later, Karas and Hillenkamp also demonstrated MALDI applied to protein ionization [@DOI:10.1021/ac00171a028].
```

It also created a controversy that the widely used method of MALDI from these two people had been overlooked, and the Nobel prize was awarded to Tanaka, whose system was rarely used[@URL:https://web.archive.org/web/20070517202246/http://cmbi.bjmu.edu.cn/news/0212/55.htm].

> In this method, the peptide sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound.

Reviewer: ```suggestion
MALDI first requires the peptide sample to be co-crystallized with a matrix molecule, which is usually a volatile, low molecular-weight, organic aromatic compound.
```

Some examples of such compounds are cyno-hydroxycinnamic acid, dihyrobenzic acid, sinapinic acid, alpha-hydroxycinnamic acid, ferulic acid etc [@URL:https://pubmed.ncbi.nlm.nih.gov/23681820/].
Subsequently, the analyte is placed in a vacuum chamber in which it is irradiated with a LASER, usually at 337nm [@DOI:10.1021/cr010375i].
This laser energy is absorbed by the matrix, which then transfers that energy along with its free protons to the co-crystalized peptides without significantly breaking them.
The matrix and co-crystallized sample generate plumes, and the volatile matrix imparts its protons to the peptides as it gets ionized first.
The weak acidic conditions used as well as the acidic nature of the matrix allows easy exchange of protons for the peptides to get ionized and fly under the electrical field in the mass spectrometer.

> These ionized peptides generally form the metastable ions, most of them will fragment quickly.

Reviewer: ```suggestion
These ionized peptides generally form the metastable ions, most of them will fragment quickly [@DOI:10.1021/ac00099a029].
```

Reviewer: is this citation appropriate?

However, it can take several milliseconds and the mass spectrometry analysis can be performed before this time.

> The peptides almost always take up a single charge and thus observed and detected as [MH]+ peaks.

Reviewer: ```suggestion
Peptides ionized by MALDI almost always take up a single charge and thus observed and detected as [MH]+ species.
```

### Electrospray Ionization

> The Electrospray Ionization (ESI) technique was developed by John Fenn and coworkers[@DOI:10.1126/science.2675315] in 1989.

Reviewer: ```suggestion
ESI was first applied to peptides by John Fenn and coworkers in 1989 [@DOI:10.1126/science.2675315].
Concepts related to electrospray Ionization (ESI) were published at least as early as 1882, when Lord Rayleigh described the number of charges that could assemble on the surface of a droplet [@DOI:10.1080/14786448208628425]. 
```

> In the electrospray ionization, reverse-phase liquid-chromatography setup is usually connected to an online nano-ESI source.

Reviewer: ```suggestion
ESI is usually coupled with reverse-phase liquid-chromatography of peptides directly interfaced to a mass spectrometer.
```

A high voltage is applied between the spraying needle and the mass spectrometer.
Through this needle, a fine mist of analyte as liquid droplets form at the tip of the needle.

> Due to voltage and pressure, the droplet bursts into plumes of ions carrying protons from the liquid phase of the chromatography to impart ions to the analytes.

Reviewer: ```suggestion
Due to voltage and pressure, the droplet bursts into plumes of ions carrying protons from the liquid phase of the chromatography to impart ions to the analytes.
<!-- TODO, explicitly describe the taylor cone and coulombic explosion -->
```
